oak-run-commons/src/test/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/pipelined/PipelinedIT.java

```diff
@@ -66,52 +69,95 @@ public class PipelinedIT {
     @Rule
     public final TemporaryFolder sortFolder = new TemporaryFolder();
 
+    private static final PathFilter contentDamPathFilter = new PathFilter(List.of("/content/dam"), List.of());
+
+    private static final int LONG_PATH_TEST_LEVELS = 30;
+    private static final String LONG_PATH_LEVEL_STRING = "Z12345678901234567890-Level_";
+
     @BeforeClass
-    public static void checkMongoDbAvailable() {
+    public static void setup() throws IOException {
         Assume.assumeTrue(MongoUtils.isAvailable());
-    }
-
-    @Before
-    public void setup() throws IOException {
+        // Generate dynamically the entries expected for the long path tests
+        StringBuilder path = new StringBuilder("/content/dam");
```

Comment:
The test case needs to add /content/dam as well.

Reply:
There are more nodes added in the file below, see in line 266:

```java
    private static final List<String> EXPECTED_FFS = new ArrayList<>(List.of(
            "/|{}",
            "/content|{}",
            "/content/dam|{}",
            "/content/dam/1000|{}",
            "/content/dam/1000/12|{\"p1\":\"v100012\"}",
            "/content/dam/2022|{}",
            "/content/dam/2022/02|{\"p1\":\"v202202\"}",
            "/content/dam/2023|{\"p2\":\"v2023\"}",
            "/content/dam/2023/01|{\"p1\":\"v202301\"}",
            "/content/dam/2023/02|{}",
            "/content/dam/2023/02/28|{\"p1\":\"v20230228\"}"
    ));
```